addons/hr_recruitment_skills/models/hr_applicant.py

```diff
@@ -45,3 +62,11 @@ def _update_employee_from_applicant(self):
             } for skill in skills_to_create])
         self.env['hr.employee.skill'].create(vals_list)
         return super()._update_employee_from_applicant()
+
+    def action_add_to_job(self):
+        self = self.with_context(just_moved=True)
+        self.job_id = self.env['hr.job'].browse(self.env.context.get('active_id'))
+        self.stage_id = self.env.ref('hr_recruitment.stage_job0')
```

Comment:
Hello @Bertrand2 
My bad, I just forgot to tell you about this change :sweat_smile:

If we don't set the stage for the new process, this specification is not useful because once the applicant is moved to the new stage then only an acknowledgment email is sent to them

If we don't want to change the stage of the applicant then we should remove this line
https://github.com/odoo/odoo/pull/147559/files#diff-ac526ada0c5ce78f444d316a739826e6cfb8995b41c19213996df8eea2b7db75R67

![2024-05-15_15-54](https://github.com/odoo/odoo/assets/120459807/1ce08a37-adcb-4e15-bd8f-a207365cc145)